### ISSUE TITLE
Anonymous functions for scribble_add_autotype_event

### ISSUE THREAD

**Issue by sohomsahaun** [JujuAdams/Scribble#116]
```js
scribble_add_autotype_event("name", function(_element, _data, _charIndex) {
	textbox_name = _data[0];
});
```
Looks very nice.

**Comment by JujuAdams** [comment]
Implemented for v6.0.12

**Comment by DragoniteSpam** [comment]
Does the runner treat anonymous functions and named functions differently? I've admittedly never tried but it would be good to know about
